internal/backends/database.go

```diff
@@ -40,6 +40,8 @@ type Database interface {
 	CreateCollection(context.Context, *CreateCollectionParams) error
 	DropCollection(context.Context, *DropCollectionParams) error
 	RenameCollection(context.Context, *RenameCollectionParams) error
+
+	DBStats(context.Context) (*DBStatsResult, error)
```

Comment:
```suggestion
	Stats(context.Context, *StatsParams) (*StatsResult, error)
```

It is a Database method, so there is no need for a `DB` prefix. And `StatsParams` should be present, even if they are empty for now.

Reply:
Thanks :+1: